### PR TITLE
Increase timeout for test

### LINKED ISSUE #4314: Occasional test timeout
On `develop`, I got the following test failure once:

```python
 Scenario: I finished viewing the details of the exercise  # kalite/coachreports/features/coachreports.feature:48
    Given I am on the detail panel                          # kalite/coachreports/features/steps/coachreports.py:58
    Given I am on the detail panel                          # kalite/coachreports/features/steps/coachreports.py:58 2.419s
    When I click on the partial colored cell                # kalite/coachreports/features/steps/coachreports.py:248
    When I click on the partial colored cell                # kalite/coachreports/features/steps/coachreports.py:248 25.796s
      Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/coachreports/features/steps/coachreports.py", line 250, in impl
          click_and_wait_for_id_to_appear(context, find_css_with_wait(context, "td.partial"), "details-panel-view")
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 130, in click_and_wait_for_id_to_appear
          return id_shown_with_wait(context, elem_wait, wait_time=wait_time)
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 217, in id_shown_with_wait
          return _shown_elem_with_wait(context, (By.ID, id_str), **kwargs)
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 283, in _shown_elem_with_wait
          lambda browser: _visibility()
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/support/wait.py", line 66, in until
          value = method(self._driver)
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 283, in <lambda>
          lambda browser: _visibility()
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 276, in _visibility
          elem = context.browser.find_element(by[0], by[1])
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 664, in find_element
          {'using': by, 'value': value})['value']
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 173, in execute
          response = self.command_executor.execute(driver_command, params)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/remote/remote_connection.py", line 349, in execute
          return self._request(command_info[0], url, body=data)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/remote/remote_connection.py", line 380, in _request
          resp = self._conn.getresponse()
        File "/home/ubuntu/.pyenv/versions/2.7.9/lib/python2.7/httplib.py", line 1073, in getresponse
          response.begin()
        File "/home/ubuntu/.pyenv/versions/2.7.9/lib/python2.7/httplib.py", line 415, in begin
          version, status, reason = self._read_status()
        File "/home/ubuntu/.pyenv/versions/2.7.9/lib/python2.7/httplib.py", line 371, in _read_status
          line = self.fp.readline(_MAXLINE + 1)
        File "/home/ubuntu/.pyenv/versions/2.7.9/lib/python2.7/socket.py", line 476, in readline
          data = self._sock.recv(self._rbufsize)
      timeout: timed out
```

Going to increase the wait time on that function, in order to avoid similar future errors _or_ if it appears the test is prone to race condition I'll fix it up.

### PR DESCRIPTION
**Pull request by MCGallaspy**
Fixes #4314. No obvious goofy race condition stuff going on there, just good ol' fashioned totally sequential waiting, but not long enough.

### CODE REVIEW

**Comment by MCGallaspy** [comment]
Another:
```
 Then the modal will dismiss                                                        # kalite/distributed/features/steps/superuser_create.py:83 60.467s
      Assertion Failed: modal not dismissed!
```

It waited ~60s even... will take a peek at this.

**Comment by MCGallaspy** [comment]
Another:

```
    When I click on the right of an exercise suggestion on the next steps card # kalite/distributed/features/steps/content_recommendation.py:22 10.303s
      Traceback (most recent call last):
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1173, in run
          match.run(runner.context)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/behave/model.py", line 1589, in run
          self.func(context, *args, **kwargs)
        File "/home/ubuntu/ka-lite/kalite/distributed/features/steps/content_recommendation.py", line 24, in impl
          element = find_css_class_with_wait(context, "content-nextsteps-topic-link")
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 185, in find_css_class_with_wait
          return _find_elem_with_wait(context, (By.CLASS_NAME, css_class), **kwargs)
        File "/home/ubuntu/ka-lite/kalite/testing/behave_helpers.py", line 249, in _find_elem_with_wait
          EC.presence_of_element_located(by)
        File "/home/ubuntu/virtualenvs/venv-2.7.9/lib/python2.7/site-packages/selenium/webdriver/support/wait.py", line 75, in until
          raise TimeoutException(message, screen, stacktrace)
      TimeoutException: Message: 
      Stacktrace:
          at FirefoxDriver.prototype.findElementInternal_ (file:///tmp/tmpgw86b5/extensions/fxdriver@googlecode.com/components/driver-component.js:10271)
          at FirefoxDriver.prototype.findElement (file:///tmp/tmpgw86b5/extensions/fxdriver@googlecode.com/components/driver-component.js:10280)
          at DelayedCommand.prototype.executeInternal_/h (file:///tmp/tmpgw86b5/extensions/fxdriver@googlecode.com/components/command-processor.js:12274)
          at DelayedCommand.prototype.executeInternal_ (file:///tmp/tmpgw86b5/extensions/fxdriver@googlecode.com/components/command-processor.js:12279)
          at DelayedCommand.prototype.execute/< (file:///tmp/tmpgw86b5/extensions/fxdriver@googlecode.com/components/command-processor.js:12221)
```

**Comment by MCGallaspy** [comment]
Success here is a false positive: https://circleci.com/gh/learningequality/ka-lite/817

File: kalite/coachreports/features/steps/coachreports.py [hunk 1]
```diff
@@ -249,7 +249,8 @@ def impl(context):
 
 @when(u"I click on the partial colored cell")
 def impl(context):
-    click_and_wait_for_id_to_appear(context, find_css_with_wait(context, "td.partial"), "details-panel-view")
+    click_and_wait_for_id_to_appear(context, find_css_with_wait(context, "td.partial"), "details-panel-view",
+                                    wait_time=30)
```
Comment (aronasorman) [inline kalite/coachreports/features/steps/coachreports.py]: Is this the main change for this PR? The rest is just cleanup?

Comment (MCGallaspy) [inline kalite/coachreports/features/steps/coachreports.py]: No, there'a also this: https://github.com/learningequality/ka-lite/pull/4316/files#diff-bde37810880ca8c8ffa2df6c7ca804d5R89

**Comment by aronasorman** [comment]
Whoops, accidentally merged this! Sorry about that. Reverted it in another PR.

**Pull request merged**